src/lib/styles/elements/breadcrumbs.css

```diff
@@ -0,0 +1,13 @@
+/* Tailwind Elements: breadcrumbs.css */
+
+@layer components {
+	.breadcrumbs {
+		@apply flex items-center space-x-4 w-full hide-scrollbar overflow-x-auto;
```

Comment:
Looks a bit unfinished on small screen sizes. A method I've used to great effect before is hiding all but the previous link on small screens? I might also suggest a shadow to show there's content being hidden- see https://css-tricks.com/books/greatest-css-tricks/scroll-shadows/ 

![image](https://user-images.githubusercontent.com/36913739/210910038-7de0922f-2227-47ca-8705-bd86e81792ff.png)

Reply:
I like your idea about shadows, but unfortunately this is not something that can be added with pure CSS. Even the CSS Tricks examples use JS to check scroll position.

Love the idea about limiting to the last two options on mobile, so I've implement exactly that:

`.breadcrumb-non-responsive` - shows all options, works as before
`.breadcrumb` - auto-responsive by default

It works really well in execution!

Oh and I dropped the "s" in `.breadcrumbs` in case you're curious. We use singular names for all other elements.